morello/morello.md

```diff
@@ -163,11 +163,12 @@ All content in this document is at the **Alpha** quality level.
 
 This document refers to, or is referred to by, the following documents.
 
-| Ref                                                                    |  URL or other reference        |  Title                                               |
-| :---                                                                   |  :---                          |  :---                                                |
-| [ACLE-morello](https://github.com/ARM-software/acle/tree/main/morello) |  This document                 |  Morello Supplement to the Arm C Language Extensions |
-| [ACLE](https://developer.arm.com/documentation/101028/latest)          |  Document number: 101028       |  Arm C Language Extensions                           |
-| [CHERI](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-947.pdf)       |  UCAM-CL-TR-947, SSN 1476-2986 |  CHERI C/C++ Programming Guide                       |
+| Ref                                                                    |  URL or other reference                              |  Title                                               |
+| :---                                                                   |  :---                                                |  :---                                                |
+| [ACLE-morello](https://github.com/ARM-software/acle/tree/main/morello) |  This document                                       |  Morello Supplement to the Arm C Language Extensions |
+| [ACLE](https://developer.arm.com/documentation/101028/latest)          |  Document number: 101028                             |  Arm C Language Extensions                           |
+| [CHERI](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-947.pdf)       |  UCAM-CL-TR-947, SSN 1476-2986                       |  CHERI C/C++ Programming Guide                       |
+| [CHERI-HYBRID](https://github.com/CTSRD-CHERI/cheri-hybrid-c-guide)    |  https://github.com/CTSRD-CHERI/cheri-hybrid-c-guide |  CHERI Hybrid C Guide                                |
```

Comment:
Suggesting this change because that's the title of the document I see in the README file at  https://github.com/CTSRD-CHERI/cheri-hybrid-c-guide

```suggestion
| [CHERI-HYBRID](https://github.com/CTSRD-CHERI/cheri-hybrid-c-guide)    |  https://github.com/CTSRD-CHERI/cheri-hybrid-c-guide |  CHERI Hybrid C/C++ Guide                                |
```